package/gluon-mesh-batman-adv-core/files/lib/gluon/upgrade/310-gluon-mesh-batman-adv-core-mesh

```diff
@@ -51,9 +51,22 @@ uci:section('network', 'interface', 'bat0',
 		    proto = 'none',
 		    macaddr = sysconfig.primary_mac,
 		    multicast_router = 2,
+		    learning = 0,
 	    }
 )
 
+uci:delete('network', 'client_lan')
+if sysconfig.lan_ifname then
+	uci:section('network', 'interface', 'client_lan',
+		    {
+			    unicast_flood = 0,
+		    }
+	)
+	for _, lanif in ipairs(lutil.split(sysconfig.lan_ifname, ' ')) do
+		uci:set('network', 'client_lan', 'ifname', lanif)
+	end
```

Comment:
This loop won't work if there are multiple LAN interfaces. Maybe you meant `add_to_set`?

Just adding `ifname = lutil.split(sysconfig.lan_ifname, ' ')` to the uci:section above seems simpler though, or am I overlooking something?